sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -32,10 +32,18 @@ Licensed under the MIT License.
     <tag>HEAD</tag>
   </scm>
 
+  <repositories>
```

Comment:
Can I get a little more context on why we are using this blob feed? If we have something that we need to consume for our builds we should try and use our Azure Devops java feed. https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-java